static/js/musicOnHover.js

```diff
@@ -0,0 +1,10 @@
+function playIt(x) {
+  var audio = document.getElementById(x);
```

Comment:
Is there a way to do it by querySelector?

Reply:
No :smirk: .

Reply:
I tried it and couldn't do it, so thought you may find.